### PR TITLE
Adding dynamic tokens and deleting GOV_DO_TOKEN

### PR DESCRIPTION
**Pull request by Faustrox**

### CODE REVIEW

**Review by RutsSantos — APPROVED** [review]

File: app/helpers/Intersect.js [hunk 1]
```diff
@@ -399,10 +399,28 @@ function notifyUserOfRisk() {
  * @param {*} url
  */
 async function retrieveUrlAsJson(url) {
-  let response = await fetch(url, {
-    method: 'GET',
-    headers: { 'Content-Type': 'application/json', gov_do_token: GOV_DO_TOKEN },
-  });
+  let GOV_DO_TOKEN = await getToken();
+
+  const responseFunc = () => {
```
Comment (D-Rosa99) [inline app/helpers/Intersect.js]: you can add the GOV_DO_TOKEN variable in the function responseFunc and pass a parameter to the function if you need it, so you can call in the if statement. and delete the variable there
```suggestion
  const responseFunc = async (validToken) => {
       const GOV_DO_TOKEN = await getToken(validToken);
```

**Review by D-Rosa99 — CHANGES_REQUESTED** [review]

**Comment by D-Rosa99** [comment]
You have this piece of code repeated in many places, you can make a utils function of this try catch and call it wherever you need it and pass it all the paremeter to work with 

image

**Review by D-Rosa99 — CHANGES_REQUESTED** [review]
You have this piece of code repeated in many places, you can make a function with the parameter you need to work and put it in a utils folders or something like that and call it wherever you need it.

https://github.com/intellisysdcorp/covid-safe-paths/blob/40e4f213f7ad19918e7c016b55cab19d95b30f2b/app/services/LocationService.js#L223-L238

**Comment by Faustrox** [comment]
@D-Rosa99 Is not the same function

File: app/helpers/Intersect.js (hunk 1 above)
Comment (Faustrox) [reply app/helpers/Intersect.js:404]: What? I didn't understand

**Comment by Faustrox** [comment]
@D-Rosa99 If I do that, then that function will have a lot of parameters, for me it isn't going to look well

**Review by D-Rosa99 — APPROVED** [review]

**Pull request merged**